onnxruntime/test/python/transformers/parity_utilities.py

```diff
@@ -41,7 +41,6 @@ def export_onnx(model, onnx_model_path, float16, hidden_size, device):
                       input_names=['input'],
                       output_names=["output"],
                       dynamic_axes=dynamic_axes,
-                      example_outputs=outputs,
```

Comment:
Will the new code continue to support pytorch 1.10 and 1.9?

Reply:
I am testing this for pytorch 1.10 now. example_outputs is not a required argument. So, it should work theoretically.

Reply:
@snn Does our CI support multiple pytorch versions? Should it?
Some pytorch changes are BC breaking, so we change to the newer versions, we have no guarantee they will work on older ones.